cmd/arcstat/arcstat.in

```diff
@@ -88,6 +88,12 @@ cols = {
     "mfug":       [4, 1000, "MFU ghost list hits per second"],
     "mrug":       [4, 1000, "MRU ghost list hits per second"],
     "eskip":      [5, 1000, "evict_skip per second"],
+    "el2skip":    [7, 1000, "evict skip, due to l2 writes, per second"],
+    "el2cach":    [7, 1024, "Size of L2 cached evictions per second"],
```

Comment:
I think these are better described as "counts" or "number" than "size"

Reply:
Of course they are.

Reply:
I was wrong, all evict_l2_* arcstats, except for skips, are expressed as the logical size of the buffers evicted. So the new types I introduced in this PR `evict_l2_eligibile_mfu` and `evict_l2_eligible_mru` are also expressed in logical size.